freertos/FreeRTOSConfig_examples_common.h

```diff
@@ -70,8 +70,12 @@
 #define configMESSAGE_BUFFER_LENGTH_TYPE        size_t
 
 /* Memory allocation related definitions. */
+#ifndef configSUPPORT_STATIC_ALLOCATION
 #define configSUPPORT_STATIC_ALLOCATION         0
+#endif
+#ifndef configSUPPORT_DYNAMIC_ALLOCATION
 #define configSUPPORT_DYNAMIC_ALLOCATION        1
+#endif
```

Comment:
I assume that things won't work if `configSUPPORT_STATIC_ALLOCATION` and `configSUPPORT_DYNAMIC_ALLOCATION` are both set to 0? Is there a use-case for having them both set to 1, or will they always be binary opposites of each other? (In which case I guess you could just do `#define configSUPPORT_DYNAMIC_ALLOCATION (! configSUPPORT_STATIC_ALLOCATION)`?)

Reply:
You can set them both to 1, it works fine.